kaldb/src/main/proto/kaldb_configs.proto

```diff
@@ -88,6 +88,9 @@ message IndexerConfig {
   string data_directory = 6;
   // Indexer server config.
   ServerConfig server_config = 7;
+  // The max_offset_delay controls by how much the indexer can be behind
+  // before it needs to create a recovery task to catch up.
+  int64 max_offset_delay = 8;
```

Comment:
This should have units of some sort in the name, as it is a numeric value. It's not obvious what this delay is without reading the code (ie, is it seconds, ms, bytes, messages, etc).